src/auro-dropdown.js

```diff
@@ -195,6 +204,9 @@ class AuroDropdown extends LitElement {
         this.trigger.addEventListener('click', handleShow);
         this.trigger.addEventListener('keydown', showByKeyboard);
       }
+    } else {
+      this.trigger.addEventListener('click', notifyTriggerClicked);
+      this.trigger.addEventListener('keydown', notifyTriggerClicked);
```

Comment:
Can we add a test for this?

Reply:
https://github.com/AlaskaAirlines/auro-dropdown/runs/6374804250?check_suite_focus=true#step:5:72

:+1: @jason-capsule42